lib/styles/editor/_menu.scss

```diff
@@ -148,29 +150,17 @@
   }
 }
 
-.neeto-editor-link-wrapper {
-  .ne-dropdown__popup {
-    padding: 8px;
-    width: 100%;
-    max-width: 400px;
-
-    .neeto-editor-link__item {
-      display: flex !important;
-      flex-direction: row;
-      justify-content: flex-start;
-      align-items: flex-start;
-      gap: 8px;
-    }
-  }
+.neeto-editor-link__item {
```

Comment:
The padding for the link dropdown menu is missing in the new version.

New:
<img width="286" alt="Screenshot 2022-12-22 at 10 28 44 PM" src="https://user-images.githubusercontent.com/24496302/209188426-705c5e1c-1ee6-441c-9894-fd26a2dbbf41.png">
Old:
<img width="283" alt="Screenshot 2022-12-22 at 10 28 49 PM" src="https://user-images.githubusercontent.com/24496302/209188434-2047cbaa-89f5-437d-85e8-551aa17666e5.png">